ui/page/livestreamSetup/view.jsx

```diff
@@ -161,237 +177,277 @@ export default function LivestreamSetupPage(props: Props) {
     return (
       <div className={'w-full flex items-center justify-between'}>
         <span>{title}</span>
-        {!hideBtn && (
+        {!hideBtn && !isMobile && (
           <Button
             button="primary"
             iconRight={ICONS.ADD}
-            onClick={() => doNewLivestream(`/$/${PAGES.UPLOAD}?type=${PUBLISH_MODES.LIVESTREAM.toLowerCase()}`)}
+            onClick={() => createNewLivestream()}
             label={__('Create or Schedule a New Stream')}
           />
         )}
       </div>
     );
   };
 
-  return (
-    <Page>
-      {/* channel selector */}
-      {!fetchingChannels && (
-        <>
-          <div className="section__actions--between">
-            <ChannelSelector hideAnon />
-          </div>
-        </>
-      )}
+  const [tab, setTab] = React.useState('Publish');
 
-      {/* livestreaming disabled */}
-      {!livestreamEnabled && (
-        <div style={{ marginTop: '11px' }}>
-          <h2 style={{ marginBottom: '15px' }}>
-            {__('This account has livestreaming disabled, please reach out to hello@odysee.com for assistance.')}
-          </h2>
+  const HeaderMenu = (e) => {
+    return (
+      <>
+        <Button
+          key={'Publish'}
+          iconSize={18}
+          label={'Publish'}
+          button="alt"
+          onClick={() => {
+            // $FlowFixMe
+            setTab('Publish');
```

Comment:
This looks correct ... I think doesn't need a `$FlowFixMe`.
There is one more below this.